resources/assets/coffee/osu_common.coffee

```diff
@@ -27,7 +27,7 @@
     _.find currentUser.friends, target_id: user_id
 
 
-  diffColour: (difficultyRating) ->
+  diffRatingColour: (difficultyRating) ->
```

Comment:
Don't agree that it should be deleted because there could be cases where this shorthand would come as handy. Taking the underline of beatmap icons as an example:

![image](https://user-images.githubusercontent.com/43639171/126268974-6a9dd143-6fbf-4037-8fa1-19280358cc93.png)

A function that just gives a CSS value for coloring use, instead of a key-value pair setting "--diff" to a color, could serve us better though.

Reply:
nothing is using it at the moment and it should be converted to typescript eventually anyway.